extensions/vercast/package.json

```diff
@@ -1,54 +1,55 @@
 {
   "$schema": "https://www.raycast.com/schemas/extension.json",
   "name": "vercast",
-  "title": "Vercel Deployments",
-  "description": "View your Vercel deployments",
+  "title": "Vercast",
```

Comment:
```suggestion
  "title": "Vercel Deployments",
```

Reply:
I was hesitant about this because it's not really just your deployments